Package.swift

```diff
@@ -1,8 +1,32 @@
+// swift-tools-version:5.0
+
 import PackageDescription
 
 let package = Package(
     name: "SwinjectAutoregistration",
+    products: [
+        .library(
+            name: "SwinjectAutoregistration",
+            targets: ["SwinjectAutoregistration"])
+    ],
     dependencies: [
-        .Package(url: "https://github.com/Swinject/Swinject.git", majorVersion: 2)
+        .package(url: "https://github.com/Swinject/Swinject.git", from: "2.6.0"),
+        .package(url: "https://github.com/Quick/Quick", from: "2.1.0"),
+        .package(url: "https://github.com/Quick/Nimble", from: "8.0.1")
+    ],
+    targets: [
+        .target(
+            name: "SwinjectAutoregistration",
+            dependencies: [
+                "Swinject",
+            ],
+            path: "Sources"),
+        .testTarget(
+            name: "SwinjectAutoregistrationTests",
+            dependencies: [
+                "Quick",
+                "Nimble",
+                "Swinject",
+            ])
```

Comment:
We probably don't want test target in the public `Package.swift` - it would force users to check out unnecessary `NImble` & `Quick` dependencies.
For running tests on linux travis environment, `.Package.test.swift` should be used (see [config](https://github.com/Swinject/SwinjectAutoregistration/blob/master/.travis.yml#L20)), although it is not configured properly now.

Reply:
Updated in 94b0ed8088b0832dab7251b129fdb972709b7492.